### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by hansven**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (3u13r) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
**Proposed solution**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (3u13r) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: Maybe we can even omit this and just use `Additional context` for this

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (3u13r) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
**Reproduction**
```

**Review by 3u13r — COMMENTED** [review]
Added some thoughts. These templates are also different from ert's. See https://github.com/edgelesssys/edgelessrt/tree/master/.github/ISSUE_TEMPLATE.
Do we want to be consistent and use those or do we want to adjust the current proposal? 
@thomasten

**Review by m1ghtym0 — APPROVED** [review]

**Pull request merged**